iroha/src/permissions.rs

```diff
@@ -265,10 +313,56 @@ impl<V: GrantedTokenValidator> PermissionsValidator for V {
     }
 }
 
+// TODO: rewrite when specialization reaches stable
```

Comment:
Can you also create task for this. Would be handy to fill even jira links near todos for easy grepping

Reply:
https://jira.hyperledger.org/browse/IR-1058